server/openapi/openapi.yaml

```
@@ -155,26 +189,39 @@ components:
       type: object
       properties:
         id:
-          type: integer
-          description: Lens profile ID.
+          oneOf:
+            - type: integer
+            - type: string
+          description: Lens profile ID, returns a hexadecimal string when `hex` is set to `true`
         handle:
           type: string
           description: Lens profile handle.
         followersCount:
           type: integer
           description: Number of followers.
+        score:
+          type: number
+          description: Profile score of a user, ranges from 0 to 1
         rank:
           type: integer
-          description: Ranking of the user, zero-based.
+          description: Ranking of a user, beginning with zero.
```

```suggestion
          description: Ranking of a user (0 is the first place).
```

(“beginning with zero” may be understood as having a literal `0` prefix.)

---

We should probably change this to 1 instead of zero.  I can't see why we would ever want to start a rank with zero